lib/rubocop/cop/sorbet/no_dynamic_contract_includes.rb

```diff
@@ -0,0 +1,127 @@
+# frozen_string_literal: true
+
+#
```

Comment:
Start with a one-liner explanation of what this cop does.

Reply:
https://github.com/Shopify/rubocop-sorbet/pull/27/files#diff-4892c3bddcbe192b3a1dd30691e5d63eR3